github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-postsubmits.yaml

```diff
@@ -72,7 +72,9 @@ postsubmits:
           value: kubevirt-bot
         command: [ "/usr/local/bin/runner.sh", "/bin/sh", "-c" ]
         args:
-        - ./automation/release.sh
+        - |
+          docker login --username "$(<"$QUAY_USER")" --password-stdin=true quay.io <"$QUAY_PASSWORD"
```

Comment:
Thanks, I didn't know that output redirection works different between shells!

I thought this form would be better as shellcheck always complains like this:
```
cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
    ^--------------^ SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
                                               ^-----------------^ SC2046: Quote this to prevent word splitting.
```

Reply:
ack, then let's keep it. :)